File: backend/Modulos/models/centroCostoUsuario.go
package models

// CentroCostoUsuario is...
type CentroCostoUsuario struct {
	SERVIDOR                    string `json:"servidor"`
	IDUSUARIO                   int    `json:"idusuario"`
	IDCENTROCOSTO               int    `json:"idcentrocosto"`
	HDGCODIGO                   int    `json:"hdgcodigo"`
	ESACODIGO                   int    `json:"esacodigo"`
	CMECODIGO                   int    `json:"cmecodigo"`
	GLOUNIDADESORGANIZACIONALES string `json:"glounidadesorganizacionales"`
	ACCION                      string `json:"accion"`
}

File: frontend/src/app/models/entity/centro-costo-usuario.ts
export class CentroCostoUsuario {
    constructor (
         public servidor          ?: string,
         public idusuario         ?: number,
         public idcentrocosto     ?: number,
         public hdgcodigo         ?: number,
         public esacodigo         ?: number,
         public cmecodigo         ?: number,
         public glounidadesorganizacionales   ?: string,
         public accion            ?: string,
         public bloqcampogrilla   ?: boolean,
         public marcacheckgrilla  ?: boolean
     ) {}
}